alchemy/src/util/telemetry/client.ts

```diff
@@ -85,7 +85,28 @@ export class TelemetryClient implements ITelemetryClient {
     if (events.length === 0) {
       return;
     }
-    const { userId, ...context } = await this.context;
+    const { userId, projectId, ...context } = await this.context;
+    const safeProjectId = projectId ?? `temp_${userId}`;
```

Comment:
will this double the number of projects if the user later runs `git init`?

Reply:
not double, just +1 because all the non-git projects would have the same id since `temp_${userId}` is consistent.

no matter what we would have ids changed between non-git and git when somebody runs `git init`, so most other solutions would be *2.

The other option to make it more stable is to have `alchemy init` run `git init` to reduce how often this happens